Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -25,7 +25,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 		void SetupLayer()
 		{
-			Layer.CornerRadius = 5;
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
+				cornerRadius = 5f; // default corner radius
```

Comment:
I know that, but it's really not an issue in this case. As stated before, the reason I didn't want to just use "smaller than" is to specifically show that I am testing for the default value. A better way would probably have been to test for `CornerRadiusProperty.DefaultValue`